examples/sw/sw.cc

```diff
@@ -218,15 +218,20 @@ auto make_sw1(const funcT& func, int block_size, const std::string &a, const std
 }
 
 int main(int argc, char* argv[]) {
-  if (argc < 6)
-  {
-    std::cout << "Usage: ./sw -n <String length> -b <block_size> <1/0 - verify>\n";
-    exit(-1);
+  int problem_size;
+  int block_size;
+  bool verify;
+
+  if (argc < 6) {
+    problem_size = 1024;
+    block_size = 32;
+    verify = false;    
+  }
+  else {
+    problem_size = std::atoi(argv[2]);
```

Comment:
Since the help output is now gone it is really hard to understand which values should be in which place of the argument list. Maybe restore the output and/or remove the parameter names (since they are not used anyway)?

Reply:
I have kept the default values and added the help output.